pkg/plugins/golang/v4/scaffolds/init.go

```diff
@@ -162,5 +162,8 @@ func (s *initScaffolder) Scaffold() error {
 		&e2e.Test{},
 		&e2e.SuiteTest{},
 		&utils.Utils{},
+		&templates.DevContainerDockerfile{},
```

Comment:
Why do we need scaffold another Dockerfile?
The project has a dockerfile already.
Why it is required